### ISSUE TITLE
current : test gns3 uri fails

### ISSUE THREAD

**Issue by zerwes** [Rosa-Luxemburgstiftung-Berlin/ansible-role-gns3#4]
```
TASK [Test gns3 uri works] *****************************************************
  fatal: [instance]: FAILED! => {"changed": false, "elapsed": 0, "msg": "Status code was -1 and not [200]: Request failed: <urlopen error [Errno 111] Connection refused>", "redirected": false, "status": -1, "url": "http://172.17.0.2:3080/static/web-ui/bundled"}
```

**Comment by zerwes** [comment]
```
root@instance:/# service gns3 status
● gns3.service - GNS3 server
     Loaded: loaded (/lib/systemd/system/gns3.service; enabled; preset: enabled)
     Active: activating (auto-restart) (Result: exit-code) since Fri 2023-07-14 08:54:54 UTC; 3s ago
    Process: 11407 ExecStartPre=/bin/mkdir -p /var/log/gns3 /var/run/gns3 (code=exited, status=0/SUCCESS)
    Process: 11408 ExecStartPre=/bin/chown -R gns3:gns3 /var/log/gns3 /var/run/gns3 (code=exited, status=0/SUCCESS)
    Process: 11409 ExecStart=/usr/bin/gns3server --log /var/log/gns3/gns3.log (code=exited, status=1/FAILURE)
   Main PID: 11409 (code=exited, status=1/FAILURE)
        CPU: 41ms
```

**Comment by zerwes** [comment]
```
Jul 14 08:57:14 instance systemd[1]: gns3.service: Scheduled restart job, restart counter is at 45.
Jul 14 08:57:14 instance systemd[1]: Stopped gns3.service - GNS3 server.
Jul 14 08:57:14 instance systemd[1]: Starting gns3.service - GNS3 server...
Jul 14 08:57:14 instance systemd[1]: Started gns3.service - GNS3 server.
Jul 14 08:57:14 instance gns3server[11657]: Traceback (most recent call last):
Jul 14 08:57:14 instance gns3server[11657]:   File "/usr/bin/gns3server", line 5, in <module>
Jul 14 08:57:14 instance gns3server[11657]:     from gns3server.main import main
Jul 14 08:57:14 instance gns3server[11657]: ModuleNotFoundError: No module named 'gns3server'
Jul 14 08:57:14 instance systemd[1]: gns3.service: Main process exited, code=exited, status=1/FAILURE
```

**Comment by zerwes** [comment]
fixed in 38da1c6
